paper/paper.md

```diff
@@ -0,0 +1,41 @@
+# CMakePPLang
+
+## Summary
+
+
+
+## Statement of Need
+
+CMake is small, extensible, and exceeds at generating build systems for many combinations of platforms, compilers, and build configurations.
+
+CMake has become the *de facto* standard tool for building C, C++, and Fortran
+programs of moderate to large size. As the size of a project increases, the
+complexity of the CMake code to build it tends to also increase. Object-
+oriented programming excels at managing large, complex code bases, and there
+is an increasing need for this in the CMake language.
```

Comment:
If you can find a reference (and we have reference slots available) it would be good to cite something here about benefits of OOP.